applications/vanilla/controllers/api/ThemesApiController.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+use Garden\Schema\Schema;
+use Garden\Web\Data;
+use Garden\Web\Exception\NotFoundException;
+use Vanilla\ApiUtils;
+use Vanilla\AddonManager;
+use Vanilla\Addon;
+use Vanilla\Models\ThemeModel;
+use Vanilla\Models\ThemeAssetModel;
+
+/**
+ * API Controller for the `/drafts` resource.
```

Comment:
```suggestion
 * API Controller for the `/themes` resource.
```

I think there was copy/paste mistake here.

Reply:
Fixed